packages/hub/node-tests/helpers/cards.ts

```diff
@@ -105,11 +106,11 @@ export function cardHelpers(mochaContext: Mocha.Suite) {
     if (!container) {
       throw new Error('Make sure cardHelpers are run after setupHub. It needs a configured container');
     }
-    currentCardService = await container.lookup('card-service');
-    fileCacheConfig = (await container.lookup('file-cache-config')) as TestFileCacheConfig;
-    fileCache = await container.lookup('file-cache');
+    currentCardService = await container.lookup('card-service', { type: 'service' });
+    fileCacheConfig = (await container.lookup('file-cache-config', { type: 'service' })) as TestFileCacheConfig;
+    fileCache = await container.lookup('file-cache', { type: 'service' });
```

Comment:
@ef4 @habdelra This pattern, along with it's counterpart in `register`, is a pain. Can we:
1. Change the APIs to first class "types" so it's like: `container.lookup('service', 'file-cache')`
1. If we dont want to change the api, at least change the default type from `default` to `service` so this isn't necessary in the default cause?

Reply:
yeah we were conflicted over this ordering as well. we set it like this because the type is optional and optional params are added to the end. i don't have any strong feelings for preserving that though...